package-res/ccc/core/cartesian/axis/abstract-cart-axis-panel.js

```diff
@@ -667,6 +667,8 @@ def
     },
 
     _calcContinuousBandSizeMin: function() {
+        if(this.chart.visualRoles.multiChart.isBound()) return 0;
```

Comment:
Think this should not be done here. This minimum band size only causes less ticks to be used.